packages/linear-progress/README.md

```diff
@@ -0,0 +1,61 @@
+> ✨ Are you a part of the Material Design web community? Help us improve by filling out this <a href='https://bit.ly/materialwebsurvey'>**10 minute survey**</a>. ✨
+
+# React Linear Progress
+
+A React version of an [MDC Linear Progress](https://github.com/material-components/material-components-web/tree/master/packages/mdc-linear-progress).
+
+## Installation
+
+```
+npm install @material/react-linear-progress
+```
+
+## Usage
+
+### Styles
+
+with Sass:
+```js
+import '@material/react-linear-progress/index.scss';
+```
+
+with CSS:
+```js
+import '@material/react-linear-progress/dist/linear-progress.css';
+```
+
+### Javascript Instantiation
+```js
+import LinearProgress from '@material/react-linear-progress';
+import React from 'react';
+
+class MyApp extends React.Component {
+  render() {
+    return (
+      <LinearProgress
+        buffer={0.8}
+        progress={0.9}
```

Comment:
I should rather swap these two around for the sample, otherwise copying this will not reveal the buffer.